Mathlib/Algebra/Category/ModuleCat/Subobject.lean

```diff
@@ -82,7 +82,7 @@ noncomputable def toKernelSubobject {M N : ModuleCat.{v} R} {f : M ⟶ N} :
 @[simp]
 theorem toKernelSubobject_arrow {M N : ModuleCat R} {f : M ⟶ N} (x : LinearMap.ker f) :
     (kernelSubobject f).arrow (toKernelSubobject x) = x.1 := by
-  -- Porting note: The whole proof was just `simp [toKernelSubobject]`.
+  -- Porting note (#10959): The whole proof was just `simp [toKernelSubobject]`.
```

Comment:
Small nit: I think English orthography ask to start in lower-case after a colon. (This is pre-existing, though.)